digits/model/images/forms.py

```diff
@@ -34,3 +30,81 @@ class ImageModelForm(ModelForm):
             tooltip = "Subtract the mean file or mean pixel for this dataset from each image."
             )
 
+    aug_flip = utils.forms.SelectField('Flipping',
+            choices = [
+                ('none', 'None'),
+                ('fliplr', 'Horizontal'),
+                ('flipud', 'Vertical'),
+                ('fliplrud', 'Horizontal and/or Vertical'),
+                ],
+            default='none',
+            tooltip = "Randomly flips each image during batch preprocessing."
+            )
+
+    aug_quad_rot = utils.forms.SelectField('Quadrilateral Rotation',
+            choices = [
+                ('none', 'None'),
+                ('rot90', '0, 90 or 270 degrees'),
+                ('rot180', '0 or 180 degrees'),
+                ('rotall', '0, 90, 180 or 270 degrees.'),
+                ],
+            default='none',
+            tooltip = "Randomly rotates (90 degree steps) each image during batch preprocessing."
+            )
+
+    aug_rot_use = utils.forms.BooleanField('Arbitrary Rotation',
+            default = False,
+            tooltip = "You can augment your dataset by performing rotation on images.",
+            validators=[ 
+                ]
+            )
+    aug_rot = utils.forms.IntegerField('Rotation (+- deg)',
+            default=45,
+            validators=[ 
+                validators.NumberRange(min=0, max=360)
+                ],
+            tooltip = "The uniform-random rotation angle that will be performed during batch preprocessing."
+            )
+
+    aug_scale_use = utils.forms.BooleanField('Random Scaling',
```

Comment:
Cropping is different: there is no value of the crop length that yields no transformation.

I'm in favour of removing the boolean field here and rely on just the `aug_scale` field instead but I'm open to discussing it :-)